symplyphysics/laws/electricity/current_is_proportional_to_EMF.py

```diff
@@ -0,0 +1,33 @@
+
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+
+# Description
+## Ohm's law for the full section of the circuit for direct current is formulated: 
```

Comment:
It's almost a duplicate of the existing Ohm's law. But for a test task it's good enough.